src/procedures/proc_wcc.c

```
@@ -0,0 +1,345 @@
+		if(SI_TYPE(v) != T_ARRAY) {
+			ErrorCtx_SetError("wcc configuration 'nodeLabels' should be an array of strings");
+			goto error;
+		}
+
+		if(!SIArray_AllOfType(v, T_STRING)) {
+			// error
+			ErrorCtx_SetError("wcc configuration 'nodeLabels' should be an array of strings");
+			goto error;
+		}
+
+		_lbls = array_new(LabelID, 0);
```

Cppcheck (2.10-2)

[error] Common realloc mistake

(memleakOnRealloc)